components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/SCIMCommonUtils.java

```diff
@@ -321,7 +321,7 @@ public static String formatDateTime(Date date) {
      */
     public static Map<String, String> getSCIMtoLocalMappings() throws UserStoreException {
 
-        String spTenantDomain = getTenantDomainFromSP();
+        String spTenantDomain = getTenantDomain();
```

Comment:
can we do the same for customSchema as well and get rid of that  `String userTenantDomain = getTenantDomainFromContext();` and use this.

Also can we rename this spTenantDomain -> tenantDomain

Reply:
Addressed with commit bb4f9622b8914f2c96b3c52ed8bb0b5373f6da3a